mem/mem_linux.go

```diff
@@ -426,3 +429,84 @@ func calcuateAvailVmem(ret *VirtualMemoryStat, retEx *VirtualMemoryExStat) uint6
 
 	return availMemory
 }
+
+const swapsFilePath = "/proc/swaps"
+
+// swaps file column indexes
+const (
+	nameCol = 0
+	// typeCol     = 1
+	totalCol = 2
+	usedCol  = 3
+	// priorityCol = 4
+)
+
+func SwapDevices() ([]*SwapDevice, error) {
+	return SwapDevicesWithContext(context.Background())
+}
+
+func SwapDevicesWithContext(ctx context.Context) ([]*SwapDevice, error) {
+	f, err := os.Open(swapsFilePath)
```

Comment:
Could you change to `common.HostProc("swaps")` like [here](https://github.com/shirou/gopsutil/blob/master/process/process_linux.go#L545)? It can enable to run on container environment.